Description: python3 puts some stuff into multiarch paths
 an alternative would be to use $(PYTHON)-config --includes
Author: Julian Taylor <jtaylor@ubuntu.com>
--- a/Demos/embed/Makefile
+++ b/Demos/embed/Makefile
@@ -3,6 +3,7 @@
 PYVERSION=$(shell $(PYTHON) -c "import sys; print(sys.version[:3])")
 
 INCDIR=$(shell $(PYTHON) -c "from distutils import sysconfig; print(sysconfig.get_python_inc())")
+PLATINCDIR=$(shell $(PYTHON) -c "from distutils import sysconfig; print(sysconfig.get_python_inc(plat_specific=True))")
 LIBDIR1=$(shell $(PYTHON) -c "from distutils import sysconfig; print(sysconfig.get_config_var('LIBDIR'))")
 LIBDIR2=$(shell $(PYTHON) -c "from distutils import sysconfig; print(sysconfig.get_config_var('LIBPL'))")
 PYLIB=$(shell $(PYTHON) -c "from distutils import sysconfig; print(sysconfig.get_config_var('LIBRARY')[3:-2])")
@@ -17,7 +18,7 @@
 	$(LINKCC) -o $@ $^ -L$(LIBDIR1) -L$(LIBDIR2) -l$(PYLIB) $(LIBS) $(SYSLIBS) $(LINKFORSHARED)
 
 embedded.o: embedded.c
-	$(CC) -c $^ -I$(INCDIR)
+	$(CC) -c $^ -I$(INCDIR) -I$(PLATINCDIR)
 
 CYTHON=../../cython.py
 embedded.c: embedded.pyx
